frontend/wallet/src/render/components/Header.re

```diff
@@ -117,6 +117,7 @@ module SyncStatus = {
       | `SYNCED => <Alert kind=`Success message="Synced" />
       | `BOOTSTRAP => <Alert kind=`Warning message="Syncing" />
       | `CONNECTING => <Alert kind=`Warning message="Connecting" />
+      | `CATCHUP => <Alert kind=`Warning message="Catching up" />
```

Comment:
Unrelated to my diff, but the schema file hadn't been regenerated

Reply:
At some point, we should revisit this, b/c as a user, Idk what is the diff between syncing, and catching up. Either we should consolidate both into syncing, or split into bootstrapping / catching up and have some explanation somewhere.